app/main.py

```diff
@@ -1,3 +1,63 @@
+from typing import Union
+
+
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: Union[float, int]) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Union["Distance", int, float]) -> "Distance":
```

Comment:
You could use `|` syntaxis for type annotation instead of `Union` and indicate the return type as `Distance` without `""` here and in other methods. Replace type annotation to utilize the latest language features.

Reply:
NameError: name 'Distance' is not defined 
I get this error when use Distance without ""

Reply:
and this 
TypeError: unsupported operand type(s) for |: 'str' and 'type'
where I use "|" with " 'Distance' "